css/style.css

```diff
@@ -0,0 +1,100 @@
+* {
+  margin: 0;
+}
+
+.container {
+  font-family: sans-serif;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 5rem 1rem;
+  margin: 0 auto;
+}
+
+#title {
+  text-align: center;
+  padding-bottom: 2.5rem;
+  font-size: 3em;
+}
+
+#description {
+  padding-bottom: 2em;
+  text-align: center;
+  line-height: 1.6em;
+}
+
+#survey-form {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  padding: 1em;
+  gap: 1.5em;
+  background-color: rgba(197, 186, 186, 0.411);
+  border-radius: 5px;
+  padding: 2em 1em;
+}
+
+#survey-form div {
+  display: flex;
+  flex-direction: column;
+  gap: 0.5em;
+}
+
+input {
+  padding: 0.5em 1em;
+}
+
+#submit {
+  background-color: rgb(0, 135, 90);
+  font-size: 1.25em;
+  font-weight: bold;
+  border: none;
+  color: #ffffff;
+  padding: 0.5em 1em;
+  border-radius: 5px;
+}
+
+fieldset {
+  border: 0;
+  padding: 0;
+}
+
+legend {
+  padding-bottom: 0.5em;
+}
+
+#submit:hover,
+#submit:focus {
+  background-color: rgb(29, 165, 120);
+}
+
+input[type="checkbox"],
+input[type="radio"] {
+  margin-right: 0.5rem;
+}
+
+@media (min-width: 600px) {
+  .container {
+    max-width: 590px;
+  }
+
+  #submit {
+    max-width: 400px;
```

Comment:
- [ ] Please reduce the size of the button, because it is too wide right now.

![image](https://user-images.githubusercontent.com/110997325/235753553-20c8f47e-c816-4922-ad5f-4a0008e76ffa.png)

Reply:
> - [ ] Well done providing a screenshot to the student as the reviewer for clarification😊. Please kindly also include an explanation, a "why", for each requested change. This is important because it helps the author to understand the issue, provides context, encourages better communication, and leads to higher-quality code.👍 
> 
> >For Example:
> _Please reduce the size of the button, because it is too wide right now. This will make the user experience...._